Graphics/GraphicsEngineWebGPU/src/EngineFactoryWebGPU.cpp

```diff
@@ -219,6 +219,14 @@ WebGPUDeviceWrapper CreateDeviceForAdapter(EngineWebGPUCreateInfo const& EngineC
 
         if (wgpuAdapterHasFeature(wgpuAdapter, WGPUFeatureName_BGRA8UnormStorage))
             Features.push_back(WGPUFeatureName_BGRA8UnormStorage);
+
+        // WGPUFeatureName_Unorm16TextureFormats = 0x000003FB
```

Comment:
Use named constants instead of magic numbers.
Add a separate file (e.g. `WebGPUStubs.h`) and move all constants there.
If they are defined in Diligent namespace, there will be no conflict if constants with the same name are added to WebGPU headers.